jetty-documentation/src/main/asciidoc/gettingstarted-guide/introduction/what-is-jetty.adoc

```diff
@@ -23,15 +23,15 @@ Eclipse Jetty is an open-source project providing an HTTP server, HTTP client, a
 
 The Jetty documentation is broken up in to four parts:
 
-* The link:{QUICKGUIDE}[Quick Start Guide] emphasizes beginning to use Jetty.
+* The link:{GSTARTEDGUIDE}[Getting Started Guide] emphasizes beginning to use Jetty.
 It provides information about what Jetty is and where you can download it, and where to find Jetty in repositories like Central Maven.
 It also provides a Quick Start guide on how to get Jetty up and running as well as an overview of how and what to configure in Jetty.
```

Comment:
I wouldn't capitalize Quick Start.

Reply:
I'd avoid the term Quick Start all together.   It is a getting started guide, so it provides a guide as to how to get started with jetty

Reply:
The Quick Start terminology has been removed. The PR changes it to Getting Started.